src/patterns/components/card/angular/info/teaser.hbs

```diff
@@ -5,107 +5,113 @@
     <col>
   </colgroup>
   <thead>
-  <tr>
-    <th>Input</th>
-    <th>Type</th>
-    <th>Description</th>
+    <tr>
```

Comment:
Is this indentation correct? Looks like it might be indented two spaces too many.